### ISSUE TITLE
Getting critical hit w/o equipping a critical hit weapon

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#235]
```
Before posting, please read the following guidelines for posts in the issue
tracker:
http://code.google.com/p/andors-trail/wiki/Forums_vs_issuetracker


What steps will reproduce the problem?
1.equip a RoLS 
2.equip an ELyR 

What is the expected output? What do you see instead?

You should now have a critical hit % of 5 & a critical hit of zero.  Now fight 
monsters until you see a message saying "you get a critical hit of X".  You 
should probably not see this as you gave no critical hit multipler.

What version of the product are you using? On what device?
0.6.9 on DROID x


Please provide any additional information below.


```

Original issue reported on code.google.com by `jgemmajr@gmail.com` on 29 Aug 2011 at 12:58

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the detailed report.

Hm, the issue is that you actually get critical hits since you have a critical 
chance. The effect is however just a normal attack since you have no critical 
modifier, so the attack damage multiplier is not applied.

This could be a case of "it's a feature, not a bug" :) But I agree that we 
should change the logic so that critical hits only can occur when you have both 
a non-zero crit chance and a non-zero crit multiplier.
```

Original comment by `oskar.wi...@gmail.com` on 30 Aug 2011 at 9:22
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r148.
```

Original comment by `oskar.wi...@gmail.com` on 30 Aug 2011 at 9:25
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 30 Aug 2011 at 9:27
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
